taskfiles/lint.yaml

```diff
@@ -762,16 +762,17 @@ tasks:
     vars:
       UV_PYTHON_PROJECTS:
         - "{{.G_INTEGRATION_TESTS_DIR}}"
+      UV_PYTHON_PROJECTS_IGNORE_ERRORS:
```

Comment:
if we agree on the approach of temporarily ignoring errors in these components, we should also create an issue **per component** to track lint compliance